internal/csi-addons/rbd/reclaimspace.go

```diff
@@ -90,12 +99,14 @@ func (rscs *ReclaimSpaceControllerServer) ControllerReclaimSpace(
 // of CSI-addons reclaimspace controller service spec.
 type ReclaimSpaceNodeServer struct {
 	*rs.UnimplementedReclaimSpaceNodeServer
+	// Embed ControllerServer as it implements helper functions
+	*rbdutil.ControllerServer
 }
 
 // NewReclaimSpaceNodeServer creates a new IdentityServer which handles the
 // Identity Service requests from the CSI-Addons specification.
-func NewReclaimSpaceNodeServer() *ReclaimSpaceNodeServer {
-	return &ReclaimSpaceNodeServer{}
+func NewReclaimSpaceNodeServer(c *rbdutil.ControllerServer) *ReclaimSpaceNodeServer {
```

Comment:
You need to use `NewNodeServer` not the ControllerServer for node level operations.

Reply:
Ahh, made typo!
Thanks, will fixed it.